### ISSUE TITLE
new polylog unicode printing missing brackets

### ISSUE THREAD

**Issue by cbm755** [sympy/sympy#25309]
Nice:
```
>>> polylog(s, z)
Liₛ(z)
```

But much room for improvement here:
```
>>> polylog(s - 1, z)
Li_s - 1(z)
```

Not quite sure what I expect in this case...  maybe it should be falling back to the <= 1.12 result of `polylog(s - 1, z)`?  There must be some precedent here in: the printing system should not need this sort of thing re-invented for each new function.

**Comment by cbm755** [comment]
Seems to be from #25004, CC @sylee957 and @PSXBRosa

**Comment by cbm755** [comment]
```python
    def _print_polylog(self, e):
        if self._use_unicode:
            func_name = f"Li_{e.args[0].__str__()}"
            new_e = Function(func_name)(e.args[1])
            return self._print_Function(new_e)
        return self._print_Function(e)
```

This doesn't look right to me, especially `__str__`.  I'm no expert on printing (who is any more?) but looking through the code I think `expint` might be a better model.

**Comment by cbm755** [comment]
I think this was fixed in 25311 and/or #25599.
